template.yml.erb

```diff
@@ -41,6 +41,146 @@ Globals:
 Conditions:
   IsDevCondition: !Equals [!Ref BaseDomainName, "dev-code.org"]
 Resources:
+  HttpDomain:
+    Type: AWS::Route53::RecordSet
+    Properties:
+      HostedZoneName: !Sub "${BaseDomainName}."
+      Name: !Sub "${SubDomainName}-http.${BaseDomainName}"
```

Comment:
Will the prod route look something like `https://javabuilderbeta-http.code.org`?

Reply:
Yes